components/benefit_sponsors/app/models/benefit_sponsors/benefit_sponsorships/benefit_sponsorship.rb

```diff
@@ -177,18 +177,37 @@ class BenefitSponsorships::BenefitSponsorship
       )
     }
 
-    scope :may_transmit_initial_enrollment?, -> (compare_date = TimeKeeper.date_of_record) {
-      where(:benefit_applications => {
-        :$elemMatch => {:"effective_period.min" => compare_date, :aasm_state => :enrollment_eligible}},
-        :aasm_state => :initial_enrollment_eligible
-      )
+    scope :may_transmit_initial_enrollment?, -> (compare_date = TimeKeeper.date_of_record, transition_at = nil) {
+      if  transition_at.blank?
+        where(:benefit_applications => {
+                  :$elemMatch => {:"effective_period.min" => compare_date, :aasm_state.in => [:enrollment_eligible, :active] }},
+              :aasm_state.in => [:initial_enrollment_eligible, :active]
```

Comment:
I see this query is also updated...which is outside the scope of the ticket. Please rever this query back to original